README.md

```diff
@@ -1136,3 +1136,13 @@ generate API documentation in the [docs](docs/) directory using
 
 Note that this script has a dependency on the `doctoc` NPM package to automate
 generating the table of contents. Install it with `npm -g i doctoc`.
+
+### Installing the Android SDK on macOS
```

Comment:
Could you add some context on who/what needs this?

Reply:
Moved the section to be immediately under [Tests](https://github.com/bazelbuild/rules_jvm_external#tests) section of the docs, so hopefully it's crystal clear.